integration-testing/src/debugDynamicAgentJpmsTest/kotlin/DynamicAttachDebugTest.kt

```diff
@@ -0,0 +1,54 @@
+@file:OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
It looks like the whole `integration-testing/src/debugDynamicAgentJpmsTest` directory is not even used at all and is just a copy of `integration-testing/jpmsTest/src`?